### PR TITLE
lsmem: Add lsmem

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

File: tests/by-util/test_lsmem.rs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// This file is part of the uutils util-linux package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+// spell-checker:ignore (words) symdir somefakedir
```
Comment (cakebaker) [inline tests/by-util/test_lsmem.rs:5]:
```suggestion
```

File: src/uu/lsmem/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[package]
+name = "uu_lsmem"
+version = "0.0.1"
+edition = "2021"
+
+[lib]
+path = "src/lsmem.rs"
+
+[[bin]]
+name = "mountpoint"
```
Comment (cakebaker) [inline src/uu/lsmem/Cargo.toml:10]:
```suggestion
name = "lsmem"
```

File: src/uu/lsmem/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+path = "src/main.rs"
+
+[dependencies]
+regex = { workspace = true }
+sysinfo = { workspace = true }
```
Comment (cakebaker) [inline src/uu/lsmem/Cargo.toml:15]: I would list only the dependencies that are currently used.
```suggestion
```

**Comment by cakebaker** [comment]
Thanks, and feel free to add some functionality ;-)

**Comment by howjmay** [comment]
thank you! I will do it right after

**Pull request merged**